### PR TITLE
[12.0][MIG] product_cost_price_avco_sync: Migration to 12.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT20698

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

File: product_cost_price_avco_sync/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        res = super(StockMove, self).write(vals)
+        if (('price_unit' in vals or 'quantity_done' in vals) and
+                not self.env.context.get('skip_avco_sync')):
+            self.cost_price_avco_sync()
+        return res
+
+    def _create_price_history(self, value):
+        PriceHistory = self.env['product.price.history']
+        for move in self:
+            PriceHistory.create({
```
Comment (Tardo) [inline product_cost_price_avco_sync/models/stock_move.py]: I think that here can use "multi style"

**Review by Tardo — CHANGES_REQUESTED** [review]

File: product_cost_price_avco_sync/models/stock_move.py (hunk 1 above)
Comment (carlosdauden) [reply product_cost_price_avco_sync/models/stock_move.py]: Changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**